frontend/public/module/k8s/types.ts

```diff
@@ -768,13 +768,35 @@ export type MachineConfigPoolKind = {
   status: MachineConfigPoolStatus;
 } & K8sResourceKind;
 
-export type ClusterUpdate = {
-  force: boolean;
-  image: string;
+export type Release = {
   version: string;
+  image: string;
+  url?: string;
   channels?: string[];
 };
 
+type PromQLClusterCondition = {
+  promql: string;
+};
+
+type ClusterCondition = {
+  type: string;
+  promql?: PromQLClusterCondition;
+};
+
+type ConditionalUpdateRisk = {
```

Comment:
no harm in typing this out, but because the console won't need to iterate over risks, it may not be necessary either.  The only bits of `ConditionalUpdate` you'll need are `release` and `conditions`.